pkg/agent/handler/credential_test.go

```diff
@@ -98,6 +106,11 @@ func TestCredentialHandler(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
+			credApp.SubResources["status"].(map[string]interface{})["level"] = prov.Success.String()
+			if tc.appStatus != "" {
```

Comment:
What are these lines for?

Reply:
Setting the status to success if an appStatus is not specified for the test, otherwise use the appStatus from the test case